app/javascript/react/components/Map/Markers/StreamMarkers.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import { AdvancedMarker, useMap } from "@vis.gl/react-google-maps";
+
+import { Session } from "./SessionType";
+import { StreamMarker } from "./StreamMarker/StreamMarker";
+
+import type { Marker } from "@googlemaps/markerclusterer";
+
+type Props = {
+  sessions: Session[];
+};
+
+const StreamMarkers = ({ sessions }: Props) => {
+  const map = useMap();
+  const [markers, setMarkers] = useState<{ [streamId: string]: Marker | null }>(
+    {}
+  );
+  const polylineRef = useRef<google.maps.Polyline | null>(null);
+
+  // Sort sessions by time
+  const sortedSessions = sessions.sort(
+    (a, b) =>
+      new Date(a.id.toString()).getTime() - new Date(b.id.toString()).getTime()
+  );
+
+  // Update markers when marker references change
+  useEffect(() => {
+    const newMarkers: { [streamId: string]: Marker | null } = {};
+    sessions.forEach((session) => {
+      if (!markers[session.point.streamId]) {
+        newMarkers[session.point.streamId] = null;
+      }
+    });
+    setMarkers((prev) => ({
+      ...prev,
+      ...newMarkers,
+    }));
+  }, [sessions]);
+
+  // Create and update polyline
+  useEffect(() => {
+    if (!map) return;
+
+    const path = sortedSessions.map((session) => ({
+      lat: session.point.lat,
+      lng: session.point.lng,
+    }));
+
+    if (polylineRef.current) {
+      polylineRef.current.setPath(path);
+    } else {
+      polylineRef.current = new google.maps.Polyline({
+        path,
+        map,
+        strokeColor: "#cee6f1",
```

Comment:
I added this color to `colors.ts` because I did it like on production and needed sth darker than `mapWater`, WDYT?
<img width="986" alt="Screenshot 2024-06-10 at 16 00 05" src="https://github.com/HabitatMap/AirCasting/assets/24482637/7b6af0de-9b8e-4f0a-9031-8f8738f5546d">